vmdb/app/models/hosts_storages.rb

```diff
@@ -1,3 +1,3 @@
 class HostsStorages < ActiveRecord::Base
-  table_name = "hosts_storages"
+  self.table_name = "hosts_storages"
```

Comment:
I could have sworn I tried it the old way but apparently I was wrong. 

```diff
diff --git a/vmdb/app/models/hosts_storages.rb b/vmdb/app/models/hosts_storages.rb
index cff1403..1d128f6 100644
--- a/vmdb/app/models/hosts_storages.rb
+++ b/vmdb/app/models/hosts_storages.rb
@@ -1,3 +1,3 @@
 class HostsStorages < ActiveRecord::Base
-  table_name = "hosts_storages"
+  table_name = "hosts_storages2"
 end
```

```
03:02:54 ~/Code/manageiq/vmdb (master) (2.1.5) + bin/rails c
Loading development environment (Rails 3.2.17)
irb(main):001:0> HostsStorages.table_name
=> "hosts_storages"
```

Should I drop this line entirely?  It looks like it never worked in the first place and the default inflector worked.